### ISSUE TITLE
Authentication URL to work on Windows & Linux

### ISSUE THREAD

**Issue by ScalaWilliam** [ActionFPS/ActionFPS-Game#80]
Connect:
actionfps://woop.ac:7654/?authid=abc&authkey=

Just load and don't connect:
actionfps:///authid=abc&authkey=

In both cases give the user confirmation in the command line that their key is loaded as well.

**Comment by ScalaWilliam** [comment]
Depends on #79.

**Comment by ScalaWilliam** [comment]
I tried the format you gave me, still didn't work.

**Comment by lucasgautheron** [comment]
The error you had indicates your client is not up to date

**Comment by ScalaWilliam** [comment]
```
william@wuxi:~/ag$ ./source/src/ac_client -t 'actionfps://authenticate?id=drakas&key=LS0tLS1CRUdJTiBEU0EgUFJJVkFURSBLRVktLS0tLQpNSUlCdWdJQkFBS.......ktLS0tLQo='
parsing commandline argument 1: "-t"
parsing commandline argument 2: "actionfps://authenticate?id=drakas&key=LS0tLS1CRUdJTiBEU0EgUFJJVkFURSBLRVktLS0tLQpNSUlCdWdJQkFBS0JnUUN5aHN5NFYzVUlUdWdPUGcvZE9Lc2MybWdOUDdwR01WSmZ4Y2F5aWZvamNuU0VwbVYvCmZHcnlOTmhsd0lqR1VzNWRVc21DOTFpbU1TcjlHYkQwbWZ0Nm4vWURvT0F
writing to file: clientlog.txt
init: sdl (1.2.15)
init: net (1.3.6)
init: world (1000)
init: video: sdl
init: video: mode
init: video: misc
init: gl
Renderer: Mesa DRI Intel(R) Haswell Mobile  (Intel Open Source Technology Center)
Driver: 3.0 Mesa 11.2.0
init: console
init: sound
Audio devices: OpenAL Soft
Sound: OpenAL Soft / OpenAL Soft (OpenAL Community)
Driver: 1.1 ALSOFT 1.16.0
init: cfg
defaultgroup = ""
init: models
init: docs
init: localconnect
failed to read private key! please issue a new key.
OpenSSL error: error:0906D06C:PEM routines:PEM_read_bio:no start line
read map packages/maps/official/ac_aqueous.cgz rev 22 (5 milliseconds)
```

**Comment by lucasgautheron** [comment]
Ok so that's a different problem now.
I'm suspecting some kind of truncation due to poor string management in AC

**Comment by lucasgautheron** [comment]
https://github.com/ActionFPS/ActionFPS-Game/commit/0a089993eb2a5fd26bb34a5925bd7ac196324492

**Comment by ScalaWilliam** [comment]
Works for me. #79 will be a separate issue to take care of.
